ftdetect/vala.vim

```diff
@@ -0,0 +1 @@
+au BufRead,BufNewFile *.vala set filetype=vala
```

Comment:
We are trying to get rid of the ftdetects; these should go into upstream. (The rest can still be merged.)

Reply:
Removed